GTFSUpdate/gtfs_files_structure.json

```diff
@@ -205,11 +209,9 @@
                     "type": "VARCHAR(255)"
                 },
                 "stop_lat": {
-                    "null": false,
                     "type": "FLOAT"
                 },
                 "stop_lon": {
-                    "null": false,
```

Comment:
How does this change get applied to the database?

Reply:
This removes the restriction that the field can't be null. We need to allow null values because the facilities in stops.txt don't have lat/lon.

Reply:
That part I get; my question is whether there's a corresponding database schema change?

Reply:
Yes there would be a database schema change.

Reply:
Is that part of this PR somewhere?

Reply:
The app GTFSUpdate that fetches and writes the GTFS data to the database uses this JSON file to create the database structure. There is no PR associated with the schema change because any change to the schema (as reflected in this file) is done automatically by the app.